htdocs/core/class/html.formfile.class.php

```diff
@@ -1958,7 +1958,10 @@ public function list_of_autoecmfiles($upload_dir, $filearray, $modulepart, $para
 		if (!empty($addfilterfields)) {
 			print '</form>';
 		}
+		return count($filearray);
 		// Fin de zone
+		
```

Comment:
Do not use empty space lines before a }.
Take a look at this doc, and enable the git hook like explained into documentation so each time you make such errors, it will be fixed automatically during the commit:  https://wiki.dolibarr.org/index.php?title=Outils_de_d%C3%A9veloppement#Installer_le_hook_git_Pre-commit